lldb/include/lldb/Expression/IRExecutionUnit.h

```diff
@@ -161,6 +162,12 @@ class IRExecutionUnit : public std::enable_shared_from_this<IRExecutionUnit>,
     return m_jitted_global_variables;
   }
 
+  void SetPreferredModules(SymbolContextList const &modules) {
```

Comment:
This is called `SetPreferredModules` but is actually `AppendToPreferredModules`.

Reply:
Also, these are no longer `modules` you should change the names to reflect that.